biomajwatcher/webapp/app/scripts/biomaj.js

```diff
@@ -64,13 +64,37 @@ angular.module('biomaj').controller('WelcomeCtrl',
     function () {});
 
 
-angular.module('biomaj').controller('biomajCtrl',
-    function ($rootScope) {
+angular.module('biomaj').controller('biomajCtrl', ['$scope', '$rootScope', '$location',
+    function ($scope, $rootScope, $location) {
         $rootScope.alerts = [];
+        $rootScope.elements = [];
         $rootScope.closeAlert = function (index) {
             $rootScope.alerts.splice(index, 1);
         };
-    });
+        $rootScope.toggleShaded = function(elementId){
+
+            $scope.elements = document.getElementsByClassName("header-label");
+            var len = $scope.elements.length;
+
+            for (var i = 0; i<len; i++){
+              if($scope.elements[i].getAttribute("id") == elementId){
+                $scope.elements[i].style.backgroundColor = '#ffffff';
```

Comment:
you should rather change/add/remove a class to the element (add or remove an *activetab* css class for example).
This would prevent hard coding some color elements in html file, which may not be a good thing when you customize CSS (using different colors, ...).